helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -155,7 +155,7 @@ spec:
 
           # Host specific resource attributes
           - name: OTEL_RESOURCE_ATTRIBUTES
-            value: host.name=$(K8S_NODE_NAME),k8s.node.name=$(K8S_NODE_NAME),k8s.pod.name=$(K8S_POD_NAME),k8s.pod.uid=$(K8S_POD_UID),k8s.namespace.name=$(K8S_NAMESPACE)
+            value: host.name=$(K8S_NODE_NAME),k8s.node.name=$(K8S_NODE_NAME)
```

Comment:
I think it's desirable to break these out into its own resource processor so that we aren't preventing straightforward usage for `OTEL_RESOURCE_ATTRIBUTES` in all deployments.

Reply:
See what you think about the update.